submissions/ses3332002/dom/script.js

```diff
@@ -0,0 +1,257 @@
+      "Speed Demon",
+      "Liberian Girl",
+      "Just Good Friends",
+      "Another Part of Me",
+      "Man in the Mirror",
+      "I Just Can't Stop Loving You",
+      "Dirty Diana",
+      "Smooth Criminal",
+      "Leave Me Alone"
+    ],
+    "bgColor": 0
+  },
+  {
+    "title": "Dangerous (1991)",
+    "cover": "dangerous",
+    "songs": [
+      "Jam",
+      "Why You Wanna Trip on Me",
+      "In the Closet",
+      "She Drives Me Wild",
+      "Remember the Time",
+      "Can't Let Her Get Away",
+      "Heal the World",
+      "Black or White",
+      "Who Is It",
+      "Give in to Me",
+      "Will You Be There",
+      "Keep the Faith",
+      "Gone Too Soon",
+      "Dangerous"
+    ],
+    "bgColor": 29
+  },
+  {
+    "title": "HIStory: Past, Present and Future, Book I (1995)",
+    "cover": "history",
+    "songs": [
+      "Scream",
+      "They Don’t Care About Us",
+      "Stranger in Moscow",
+      "This Time Around",
+      "Earth Song",
+      "D.S.",
+      "Money",
+      "Come Together",
+      "You Are Not Alone",
+      "Childhood",
+      "Tabloid Junkie",
+      "2 Bad",
+      "HIStory",
+      "Little Susie",
+      "Smile"
+    ],
+    "bgColor": 332
+  },
+  {
+    "title": "Invincible (2001)",
+    "cover": "invincible",
+    "songs": [
+      "Unbreakable",
+      "Heartbreaker",
+      "Invincible",
+      "Break of Dawn",
+      "Heaven Can Wait",
+      "You Rock My World",
+      "Butterflies",
+      "Speechless",
+      "2000 Watts",
+      "You Are My Life",
+      "Privacy",
+      "Don't Walk Away",
+      "Cry",
+      "The Lost Children",
+      "Whatever Happens",
+      "Threatened"
+    ],
+    "bgColor": 215
+  },
+  {
+    "title": "Michael (2010)",
+    "cover": "michael",
+    "songs": [
+      "Hold My Hand",
+      "Hollywood Tonight",
+      "Keep Your Head Up",
+      "(I Like) The Way You Love Me",
+      "Monster",
+      "Best of Joy",
+      "Breaking News",
+      "(I Can't Make It) Another Day",
+      "Behind the Mask",
+      "Much Too Soon"
+    ],
+    "bgColor": 8
+  },
+  {
+    "title": "Xscape (2014)",
+    "cover": "xscape",
+    "songs": [
+      "Love Never Felt So Good",
+      "Chicago",
+      "Loving You",
+      "A Place with No Name",
+      "Slave to the Rhythm",
+      "Do You Know Where Your Children Are",
+      "Blue Gangsta",
+      "Xscape"
+    ],
+    "bgColor": 33
+  }
+];
+
+let doc = document.body;
+let navTab = document.querySelector(".navigation");
+let infoTab = document.querySelector(".info");
+let navTabButtonsArray;
+
+function fillNavTab() {
+  navTab.innerHTML = "";
+  let navTabButton;
+  for (let i = 0; i < albums.length; i++) {
+    navTabButton = document.createElement("button");
+    navTabButton.classList.add("navigation_button");
+    navTabButton.innerText = albums[i].title;
+    navTab.append(navTabButton);
+  };
+  navTabButtonsArray = Array.from(document.querySelectorAll(".navigation_button"));
+};
+
+function fillInfoTab(e) {
```

Comment:
```suggestion
function fillInfoTab({ target }) {
```

You сan use ES6 destructuring here. It's 2020!

Reply:
Ready